marketplace_confg/chart/dicom-adapter/templates/manifest.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: "{{ .Release.Name }}-adapter"
+  namespace: "{{ .Release.Namespace }}"
+  labels: &DicomDeploymentLabels
+    app.kubernetes.io/name:  "{{ .Release.Name }}"
+    app.kubernetes.io/component: dicom-adapter-app
+spec:
+  replicas: 1
+  serviceName: {{ .Release.Name }}-dicom-adapter-lb
+  selector:
+    matchLabels: *DicomDeploymentLabels
+  template:
+    metadata:
+      labels: *DicomDeploymentLabels
+    spec:
+      containers:
+        - name: dicom-import-adapter
+          image: "{{ .Values.import.image.repo }}:{{ .Values.import.image.tag }}"
+          ports:
+            - containerPort: 2575
+              protocol: TCP
+              name: "port"
+          args: 
+            - "--dimse_aet={{ .Values.import.dimse_aet }}"
+            - "--dimse_port=2575"
+            - "--dicomweb_address={{ .Values.import.dicomweb_address }}"
+            {{- if .Values.import.redact_remove_list }}
+            - "--redact_remove_list={{ .Values.import.redact_remove_list }}"
+            {{- end }}
+            {{- if .Values.import.redact_keep_list }}
+            - "--redact_keep_list={{ .Values.import.redact_keep_list }}"
+            {{- end }}
+            {{- if .Values.import.dimse_cmove_aet }}
+            - "--dimse_cmove_aet={{ .Values.import.dimse_cmove_aet }}"
+            {{- end }}
+            {{- if .Values.import.aet_dictionary_inline }}
+            - "--aet_dictionary_inline={{ .Values.import.aet_dictionary_inline }}"
+            {{- end }}
+            {{- if .Values.import.destination_config_inline }}
+            - "--destination_config_inline={{ .Values.import.destination_config_inline }}"
+            {{- end }}
+            {{- if .Values.import.store_compress_to_transfer_syntax }}
+            - "--store_compress_to_transfer_syntax={{ .Values.import.store_compress_to_transfer_syntax }}"
+            {{- end }}
+            {{- if .Values.import.monitoring_project_id }}
+            - "--monitoring_project_id={{ .Values.import.monitoring_project_id }}"
+          env:
+            - name: ENV_POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: ENV_POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: ENV_CONTAINER_NAME
+              value: dicom-import-adapter
+            {{- end }}
+        {{- if .Values.export.hasexporter }}
```

Comment:
Were you able to test this to verify this config works?

Reply:
Yes, I tested the stack driver and it logged correct events.